ts/Core/Utilities.ts

```diff
@@ -515,6 +516,8 @@ function error(
         message += additionalMessages;
     }
 
+    // Display error on the chart causing the error or the last created chart.
+    chart = chart || find(charts.slice().reverse(), (c?: Chart): boolean => !!c);
```

Comment:
Can we do this logic in the debugger module instead? Because when the module is not loaded we show the error in the console, and we don't need the chart context.

Reply:
@TorsteinHonsi 
`chart` is needed in this function context only to fire the event. To keep the debugger module running on an event we would have to fire the event on something else and pass `chart` as an additional argument - so it will still be in this context. The "something else" to fire the event on could be the `Highcharts` instance - this makes some sense because if there is no chart then we loop the `charts` array, but I haven't seen any event fired like this. An alternative event source that I can think of is `window` or `document`.

Reply:
Yes, I think events that are not related to a chart should be called on one of those objects. `Highcharts`, `window` or `document`. We probably cannot use the `Chart` class either because we can have a pure renderer setup without a `Chart`.

@bre1470 Which object do you think makes more sense? When we get rid of the namespace, we still have Highcharts as the return for Globals.ts, right? And `Highcharts.win` should also always be given.

Reply:
Yes, we can use the Globals module for that purpose, but we could also consider a chart registry similar to one for series.

Reply:
We have `Highcharts.charts`. So if we pass `Highcharts` (=Globals)  to the debugger module, it will use Highcharts.charts to find the last one.

Reply:
Sorry, I confused instance and factory/constructor.

Reply:
@bre1470 I changed the context of the event to `Highcharts` to provide more info (in the commit below). If `Highcharts` is ok, then all should be done.